src/components/cms/SkillFeedbackPage/SkillFeedbackPage.js

```diff
@@ -105,6 +113,10 @@ const EnabledButtonStyles = css`
   cursor: pointer;
 `;
 
+const AuthorName = styled.span`
```

Comment:
Use either use class names or styled-components.

Reply:
Could not understand what do you mean?

Reply:
In this, I have used styled-components